src/main/java/org/dita/dost/writer/KeyrefPaser.java

```diff
@@ -384,17 +384,17 @@ private Attributes processElement(final Attributes atts) {
                 if (keyDef != null) {
                     if (currentElement != null) {
                         final NamedNodeMap attrs = elem.getAttributes();
-                        final URI target = keyDef != null ? keyDef.href : null;
+                        final URI target = keyDef.href;
```

Comment:
I removed the null check here because there's already a null check for `keyDef` on l. 384, so I think this is unnecessary, right?